### PR TITLE
usagov-617-Put-an-attribute-with-the-originating-Node-ID-within-each-…

### PR DESCRIPTION
**Pull request by sinchagrenchik**
…agency-accordion-on-the-A-to-Z-Letter-pages: Update views-view-fields--federal-agencies.html.twig

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-617

## Checklist for the Developer

- [x ] A link to the JIRA ticket has been added above.
- [ ] The JIRA ticket identifies the desired result of this change.
- [ ] The JIRA ticket contains clear acceptance criteria.
- [ ] The JIRA ticket contains clear testing steps.
- [ ] The JIRA ticket contains a description of "Done".
- [ ] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [x ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

File: web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig [hunk 1]
```diff
@@ -37,7 +37,8 @@
 #}
 
 {% set lang = row._entity.langcode.value %}
-{% set UID = 'dr-' ~ fields.title.content|striptags|trim %}
+{% set Title = 'dr-' ~ fields.title.content|striptags|trim %}
```
Comment (IsabelLaurenceau) [inline web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig:40]: Can we change this to 
{% set UTitle = 'dr-' ~ fields.title.content|striptags|trim %}

Comment (IsabelLaurenceau) [reply web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig:40]: Button needs to be changed to still use UTitle and set UID in data attribute
```
  <button
      aria-controls="{{- UTitle -}}"
      data-node-id="{{- UID -}}
      aria-expanded="false"
      class="usa-accordion__button"
      >
```

Comment (IsabelLaurenceau) [reply web/themes/custom/usagov/templates/views-view-fields--federal-agencies.html.twig:40]: Div on line 54 needs to be changed to 
`<div class="usa-accordion__content usa-prose" id="{{- UTitle -}}" data-node-id="{{- UID -}}">
`

**Review by IsabelLaurenceau — CHANGES_REQUESTED** [review]

**Comment by akf** [comment]
I'm picking this one up and starting a new branch from the current dev branch.

**Pull request closed without merging**